CHANGES.txt

```diff
@@ -202,6 +202,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       but was not applied to other dialects, and e2e tests explicitly checked
       that FORTRANFLAGS did not propagate outside the FORTRAN dialect,
       so the conclusion is that behavior is intentional (issue #2257)
+    - Improvements to lex and yacc tools: better documentation of
```

Comment:
add which flags are interpretted correctly now? also add blurb to RELEASE.txt?

Reply:
The only behavioral change is in `Tool/lex.py`: the previous lex PR left us with a list-of-lists after the change from subst to subst_list on `LEXFLAGS`, which meant we couldn't iterate over the results.  That's not material for RELEASE - the briefly broken bit was never visible in an SCons release.  I don't know if you want that mentioned in CHANGES: fix interpretation of `LEXFLAGS` in one very specific circumstance (previously untested) which was broken for a few days. Let me know...